coverage_reports.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
```

Comment:
hmm why do we need this html file ? or even the coverage_reports folder at all ? (and we even .gitignore the coverage folder ?)

Reply:
I added the generation of HTML to the coverage reports, which allows some very nice browsing of coverage per file. However, the HTML is located in the package specific ‘coverage’ folder.

That’s why I added this index to the packages. I thought you may have questions about this. 😃 

Can you try out generating the reports, and browse a little?

Reply:
haha I'll take it for a spin locally to generate reports and take a look tomorow then !